pkg/keys/keystore.go

```diff
@@ -25,14 +25,15 @@ import (
 
 	"github.com/golang/glog"
 	"github.com/pkg/errors"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
is there a reason for this change? this is the same package name

Reply:
His IDE automatically does that, I wondered the same a while ago.